### DISCUSSION TITLE
Default Fallback hotspot password

### DISCUSSION THREAD

**Discussion by Carltk** [PricelessToolkit/ESPBell-LITE#6] [General]
What is the default  Fallback hotspot password for the ESPBell-lite? 

The Yaml file shows "password" but that doesn't seem to be working 
 for my new (out of the box) device - would it be anything else?

**Comment by PricelessToolkit** [comment]
Hi @Carltk , it can't be other things than "password", but it is strange, can you use another browser to configure it?

**Reply by Carltk** [reply]
Thanks for the response.

The unit was behaving a bit strangely, the Fallback hotspot would appear then disappear and I could not get a connection to the embedded sever (192.168.4.1). I did try "password" - perhaps that should be added to the README.

I've checked the voltages and I'm getting 3.25V at the module power pins so that seems reasonable.
Is it correct that the module's blue light will flash briefly at powerup and then stay dark thereafter?

I'm going to try reloading the ESPHome code into the unit, I just need to dig out a programmer.

**Reply by PricelessToolkit** [reply]
Yes, 3.25V is OK, and yes blue light will flash briefly at powerup and then stay dark thereafter. 

It's strange that you can't connect to an embedded server! another client said the same thing! Maybe it's a ESPhome version thing; I will check it tomorrow, and I will report back.

"perhaps that should be added to the README." i will do it!! Thanks

**Reply by PricelessToolkit** [reply]
I can confirm that the Fallback hotspot does not activate every time after a disconnect; it functioned correctly 6-7 times out of 10. This behavior is consistent with the 'regular D1 mini' too. Additionally,  when it does work and incorrect WiFi credentials are provided, sometimes it fails to recover. It seems likely that the AP component of ESPhome is not stable. Unfortunately, I don't recall the specific version of ESPhome installed in the ESPBell during production, as I updated ESPhome twice.

**Reply by Carltk** [reply]
OK, I managed to connect to the device and add an SSID & Password. The device then successfully connected to the WiFi network. 

I can't say for sure what the final fix was as I have been playing around with it a bit but the last thing I tried before successful connection was putting the board through a reflow cycle in a soldering oven (drastic, I know but it was a bit of a hail-mary move as the board had become unresponsive to esptool). 

Thanks for your support.

**Reply by PricelessToolkit** [reply]
@Carltk 100% is not a soldering problem, because I test each unit manually with the home assistant on my test network, But yes I can confirm Fallback hotspot is not stable even with esp dev boards too.

**Comment by Carltk** [comment]
Hi,
I appreciate that, as I said, I did a fair bit of messing around with it and managed to kill it (no led flash on boot, no response to ESPTool).

I was having massive problems connecting to the fallback AP at home, maybe due to the high number of networks visible??? 

When I tried at work (after I fixed it), and there were fewer visible networks, connection went OK. I ended up programming my home network SSID & password into the device at work and it's been rock solid once I plugged it in at home.

I'm now up and running and absolutely loving ESPHome - what an amazing platform! I have the board sending and receiving MQTT messages to my NodeRed backend and I'll get it wired into my intercom sometime soon.

Thanks for the support.

Cheers,
Carl.
